openmp/runtime/src/kmp_platform.h

```diff
@@ -164,6 +164,9 @@
 #define KMP_ARCH_PPC_XCOFF 1
 #undef KMP_ARCH_PPC
 #define KMP_ARCH_PPC 1
+#elif defined(__powerpc__) && defined(KMP_OS_NETBSD)
```

Comment:
Instead of checking for KMP_OS_NETBSD since this really should not be NetBSD specific it would be better to check !__LP64__ since __powerpc__ is defined even on powerpc64.

Also it would be good to have #define KMP_ARCH_PPC 0 further up around line 110.